Firestore/Source/Public/FIRSource.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+/**
+ * An enum that configures the behavior of `DocumentReference.getDocument()`
+ * and `CollectionReference.getDocuments()`.  By providing a source enum the
+ * `getDocument[s]` methods can be configured to fetch results only from the
+ * server, only from the local cache, or attempt the server and fall back to
+ * the cache (which is the default).
+ *
+ * Setting the source to FIRSourceDefault, if online, causes Firestore to try
```

Comment:
Since we're using Swift syntax above, this should perhaps be Source.default ?  [in general we're really inconsistent about Swift vs Obj-C in our comments, but we may as well be consistent within a single comment block. :-)]